src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -420,6 +422,18 @@ const std::map<cmp_enum, int> cmpToIndex = {
                     throw std::invalid_argument("Actionx: " + actx.name());
                 }
 
+                /* item[15] is a parameter that indicates whether left_paren or right_paren is used in an expression
```

Comment:
What about: `(( WWCT '*' > 1)`?

Reply:
I am not sure what this comment is about - could not find it in the test-data. If these input data are used - it will never be "true".